src/generic/Router.ts

```diff
@@ -83,6 +84,17 @@ export class GenericWebhooksRouter {
         const router = Router();
         router.all(
             '/:hookId',
+            helmet({
+                contentSecurityPolicy: {
+                    useDefaults: true,
+                    directives: {
+                        default: 'self',
```

Comment:
This  renders a CSP containing `defautl self;` which is not valid AFAIK. Did you mean to spell out `default-src`?
It's set to `'self'` in the helmet default but I see why you want to avoid unexpected changes.

```suggestion
                        defaultSrc: "'self'",
```